### ISSUE TITLE
Checks to Dockerhub return a 406

### ISSUE THREAD

**Issue by EagleoutIce** [becheran/mlc#95]
When we check links to `https://hub.docker.com/r/<...>` we get a `406 - Not Acceptable` even though the links exist and work fine.

**Comment by becheran** [comment]
@EagleoutIce can you give me one example URL with `/r`? I guess the 406 means that the HEAD and later GET request is not accepted by the endpoint.

**Comment by EagleoutIce** [comment]
Sure! Specifically for us, <https://hub.docker.com/r/eagleoutice/flowr> fails.

**Comment by becheran** [comment]
@EagleoutIce I am not able to reproduce the issue. I added `[foo](https://hub.docker.com/r/eagleoutice/flowr)` in a markdown file and `mlc` did not detect any error:

`[ OK ] .\README.md:3:1 => https://hub.docker.com/r/eagleoutice/flowr -`

**Comment by EagleoutIce** [comment]
Yes, thank you very much for checking, it appears that the problem vanished automagically.

**Comment by EagleoutIce** [comment]
Ok, maybe this is a flakey github-actions issue, but we got the error again (https://github.com/flowr-analysis/flowr/actions/runs/10059706810/job/27805541854?pr=868#step:10:371)

**Comment by becheran** [comment]
I also encountered this issue now from time to time. Not sure if you are still using mlc. If you do, you could give version 0.20.0 a try. I am not fully sure if it is the case, but I have the feeling that changing the accept header in the request helped to elimitate the sporadic failure. But since it only happended sporadically I am still not 100% sure. Please re-open if it is still failing sometimes.
